clients/bigquery/storagewrite_test.go

```diff
@@ -101,6 +101,58 @@ func TestEncodePacked64TimeMicros(t *testing.T) {
 	assert.Equal(t, int64(1<<32+1000), encodePacked64TimeMicros(epoch.Add(time.Duration(1)*time.Hour+time.Duration(1)*time.Millisecond)))
 }
 
+func TestEncodePacked32TimeSeconds(t *testing.T) {
+	epoch := time.Date(0, 0, 0, 0, 0, 0, 0, time.UTC)
+
+	assert.Equal(t, int32(0), encodePacked32TimeSeconds(epoch))
+	assert.Equal(t, int32(1), encodePacked32TimeSeconds(epoch.Add(time.Duration(1)*time.Second)))
+	assert.Equal(t, int32(1<<6), encodePacked32TimeSeconds(epoch.Add(time.Duration(1)*time.Minute)))
+	assert.Equal(t, int32(1<<12), encodePacked32TimeSeconds(epoch.Add(time.Duration(1)*time.Hour)))
+	assert.Equal(t, int32(1<<12+1), encodePacked32TimeSeconds(epoch.Add(time.Duration(1)*time.Hour+time.Duration(1)*time.Second)))
+}
+
+func TestEncodePacked64DatetimeSeconds(t *testing.T) {
+	_ts := time.Date(2024, 10, 24, 13, 1, 2, 3000000, time.UTC)
```

Comment:
Nit: Do these need to be prefixed with an underscore?

Reply:
Nope, can change